### PR TITLE
Replace `ref` with `source_url`

### PR DESCRIPTION
**Pull request by coryodaniel**
Replace `ref` with `source_url`. This is a breaking change, but ... lets get it out so the docs are right :D

### CODE REVIEW

File: pkg/template/render_test.go [hunk 1]
```diff
@@ -97,14 +97,15 @@ func walkAndCompare(wantDir string, gotDir string) {
...
 
-			fmt.Printf("Comparing (want) %s and (got) %s\n", path, gotFilePath)
-
-			dmp := diffmatchpatch.New()
 			gotText, _ := readFile(gotFilePath)
 			wantText, _ := readFile(path)
-			diffs := dmp.DiffMain(wantText, gotText, false)
-
-			fmt.Println(dmp.DiffToDelta(diffs))
+			if gotText != wantText {
```
Comment (WillBeebe) [inline pkg/template/render_test.go:102]: 👍

File: pkg/template/testdata/application-templates/renders-successfully/massdriver.yaml [hunk 2]
```diff
@@ -11,20 +11,9 @@ params:
       type: string
     app_env:
       type: string
-<md if not .Connections md>
```
Comment (WillBeebe) [inline pkg/template/testdata/application-templates/renders-successfully/massdriver.yaml:14]: Interesting that the test passed with this removed. The test-case has a `draft_node` and the expected file does as well.

https://github.com/massdriver-cloud/massdriver-cli/pull/118/files#diff-26cd67a32b447e457298d13f30e1a4206249ff485a89b83029954cb7cdb16a65R15-R20

**Review by WillBeebe — APPROVED** [review]
LGTM

File: pkg/template/testdata/application-templates/renders-successfully/massdriver.yaml (hunk 2 above)
Comment (coryodaniel) [reply pkg/template/testdata/application-templates/renders-successfully/massdriver.yaml:14]: The test never added connections so this just ended up being a red herring of whitespace

**Pull request merged**